modules/ipc/ipcProviderBackend.js

```diff
@@ -26,6 +26,7 @@ const ERRORS = {
     METHOD_TIMEOUT: {"code": -32603, "message": "Request timed out for method  \'__method__\'."},
     TX_DENIED: {"code": -32603, "message": "Transaction denied"},
     BATCH_TX_DENIED: {"code": -32603, "message": "Transactions denied, sendTransaction is not allowed in batch requests."},
+    BATCH_COMPILE_DENIED: {"code": -32603, "message": "Compilation denied, compileSolidity is not allowed in batch requests."},
```

Comment:
This wasn't necessary for the fix but I added it anyway. You can't do a contract compilation as part of a batch call because we don't support that.